src/main/java/BasicJavaHomeWork/Exercise_4.java

```diff
@@ -0,0 +1,41 @@
+package BasicJavaHomeWork;
+
+import java.util.Scanner;
+
+public class Exercise_4 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Please enter a number: ");
+        int numb = in.nextInt();
+        int b;
```

Comment:
Please rename the variables with a more consistent name, for example: sum, counter, etc. Refer also to the naming convention presented yesterday

Reply:
done